package.json

```diff
@@ -1,12 +1,22 @@
 {
   "name": "klaviyo-react-native-sdk",
-  "version": "1.1.1",
+  "version": "1.2.0-alpha.1",
```

Comment:
I'm sure you've thought about it but checkin in just to be sure - there is nothing breaking with these release correct?

Reply:
Shouldn't be, but version number is definitely TBD until we've finalized our regression testing